lib/valkyrie/persistence/fedora/metadata_adapter.rb

```diff
@@ -10,40 +10,67 @@ module Valkyrie::Persistence::Fedora
   #   )
   class MetadataAdapter
     attr_reader :connection, :base_path, :schema
+
+    # @param connection
+    # @param base_path [String]
+    # @param schema [Valkyrie::Persistence::Fedora::PermissiveSchema]
     def initialize(connection:, base_path: "/", schema: Valkyrie::Persistence::Fedora::PermissiveSchema.new)
       @connection = connection
       @base_path = base_path
       @schema = schema
     end
 
+    # Construct the query service object using this adapter
+    # @return [Valkyrie::Persistence::Fedora::QueryService]
     def query_service
       @query_service ||= Valkyrie::Persistence::Fedora::QueryService.new(adapter: self)
     end
 
+    # Construct the persister object using this adapter
+    # @return [Valkyrie::Persistence::Fedora::Persister]
     def persister
       Valkyrie::Persistence::Fedora::Persister.new(adapter: self)
     end
 
+    # Generate the ID for this adapter
+    # Currently, this is an MD5 hash of the URL to the Fedora API endpoint
+    # e.g. http://localhost.localdomain:8080/fedora => 0fb9f0605dcb2f1715d2d47fa68e7046
```

Comment:
I don't think we should include the implementation details of the ID in the comment — just saying it's based on the Fedora baseURL seems like enough to me.